tcms/testcases/templates/testcases/mutable.html

```diff
@@ -100,6 +100,61 @@
+                <div>
+                    <label class="col-md-1 col-lg-1" >{% trans "Setup Duration" %}</label>
+                        <div class="col-md-3 col-lg-3" {% if form.setup_duration.errors %}has-error{% endif %}>
+                            <div class="col-md-3 col-lg-3">
+                                 <input id="setup-duration-days" type="text" class="form-control" value="{{ form.setup_duration.value }}">
+                                 <label>day</label>
+                            </div>
+
+                            <div class="col-md-3 col-lg-3">
+                                <input id="setup-duration-hours" type="text" class="form-control" value="{{ form.setup_duration.value }}">
+                                <label>hours</label>
+                            </div>
+
+                            <div class="col-md-3 col-lg-3">
+                                <input id="setup-duration-minutes" type="text" class="form-control" value="{{ form.setup_duration.value }}">
+                                <label>minutes</label>
+                             </div>
+
+                            <div class="col-md-3 col-lg-3">
+                                <input id="setup-duration-seconds"  type="text" class="form-control" value="{{ form.setup_duration.value }}">
+                                <label>seconds</label>
+                            </div>
+                        </div>
+                        {{ form.setup_duration.errors }}
+                        {% include "include/duration_picker_script.html" with selector="#setup-duration" %}
+
+                </div>
+                    <label class="col-md-1 col-lg-1" >{% trans "Testing Duration" %}</label>
+                        <div class="col-md-3 col-lg-3" {% if form.testing_duration.errors %}has-error{% endif %}>
+                            <div class="col-md-3 col-lg-3">
+                                 <input id="testing-duration-days" type="text" class="form-control" value="{{ form.testing_duration.value }}">
+                                 <label>days</label>
+                            </div>
+
+                            <div class="col-md-3 col-lg-3">
+                                <input id="testing-duration-hours"  type="text" class="form-control" value="{{ form.testing_duration.value }}">
+                                <label>hours</label>
+                            </div>
+
+                            <div class="col-md-3 col-lg-3">
+                                <input id="testing-duration-minutes" type="text" class="form-control" value="{{ form.testing_duration.value }}">
+                                <label>minutes</label>
+                             </div>
+
+                            <div class="col-md-3 col-lg-3">
+                                <input id="testing-duration-seconds" type="text" class="form-control" value="{{ form.testing_duration.value }}">
+                                <label>seconds</label>
```

Comment:
Passing the same value for multiple widgets of the same field is probably not going to work! However full testing of this is blocked by the missing javascript portion for this widget.